integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/apim/utils/PublisherUtils.java

```diff
@@ -109,6 +114,24 @@ public static String createAPIUsingOAS(JSONObject apiProperties,
         return apiDTO.getId();
     }
 
+    public static String updateOpenAPIDefinition(String apiId, String openAPIFileName,
```

Comment:
Shall we add documentation to this new method ?

Reply:
Fixed with https://github.com/wso2/product-microgateway/pull/2646/commits/6c3651005ec5a97c5b9831526bb82b0b9fce45d9